assets/src/edit-story/components/canvas/framesLayer.js

```diff
@@ -67,6 +67,18 @@ const Hint = styled.div`
   background-color: ${({ theme }) => theme.colors.bg.workspace};
 `;
 
+const DesignSpaceGuideline = styled.div`
+  border: 1px solid ${({ theme }) => theme.colors.callout};
+  left: ${100 * (48 / 412)}%;
+  right: ${100 * (48 / 412)}%;
```

Comment:
Let's use constants here for better readability.

Reply:
Done in 6eaf4e4e1.